### PR TITLE
[release-1.2] Move hotplug setups to webhook

### PR DESCRIPTION
**Pull request by xpivarc**
### What this PR does
Moves the hotplug defaulting to webhook in order to properly work with defaulting of cpu topology. Without this path a VM without topology, with request &/ limits would most likely fail to start.

Fixes #

Links to places where the discussion took place: 

### Special notes for your reviewer
This is 1.2 specific regression. Therefore I skip main.

### Release note

```release-note
Virtual Machines without topology can start with live updates enabled
```

### CODE REVIEW

File: tests/hotplug/cpu.go [hunk 1]
```diff
@@ -61,6 +62,24 @@ var _ = Describe("[sig-compute][Serial]CPU Hotplug", decorators.SigCompute, deco
...
+
+		It("should be able to start", func() {
+
+			vmi := libvmi.NewAlpineWithTestTooling(
+				libvmi.WithMasqueradeNetworking()...,
+			)
+			vmi.Namespace = testsuite.GetTestNamespace(vmi)
+			// This is more than default maxSockets = 4
+			vmi.Spec.Domain.Resources.Requests[kubev1.ResourceCPU] = resource.MustParse("5000m")
+			vm := libvmi.NewVirtualMachine(vmi, libvmi.WithRunning())
```
Comment (Barakmor1) [inline tests/hotplug/cpu.go:75]: Please use WithRunStrategy

**Comment by xpivarc** [comment]
@Barakmor1 Thanks for the review. I already opened https://github.com/kubevirt/kubevirt/pull/12583 for main and 1.3 backport.

**Pull request merged**